test/PolyhedralGeometry/polyhedral_complex.jl

```diff
@@ -151,4 +151,28 @@
       @test n_maximal_polyhedra(vrep) == n_maximal_polyhedra(hrep)
     end
   end
+
+  @testset "Operators" begin
+    PC = polyhedral_complex(normal_fan(cube(2)))
+    PCshifted = PC + [1, 1]
```

Comment:
Test some more examples:
```suggestion
  @testset "Transformations" for P in (cube(2), dodecahedron(), n_gon(5))
    PC = polyhedral_complex(normal_fan(P))
    PCshifted = PC + fill(1, dim(P))
```
This uncovered the rays issue below, and these three objects cover the most important types that we support (QQ, number fields, QQBar).